src/api/ledger/parse/orderbook-order.js

```diff
@@ -4,22 +4,25 @@ const utils = require('./utils');
 const flags = utils.core.Remote.flags.offer;
 const parseAmount = require('./amount');
 
+function replaceValue(amount, value) {
+  return _.assign({}, amount, {value});
+}
+
 function parseOrderbookOrder(order: Object): Object {
   const direction = (order.Flags & flags.Sell) === 0 ? 'buy' : 'sell';
   const takerGetsAmount = parseAmount(order.TakerGets);
   const takerPaysAmount = parseAmount(order.TakerPays);
   const quantity = (direction === 'buy') ? takerPaysAmount : takerGetsAmount;
   const totalPrice = (direction === 'buy') ? takerGetsAmount : takerPaysAmount;
+  const price = replaceValue(totalPrice, (direction === 'buy') ?
+    utils.invertQuality(order.quality) : order.quality);
```

Comment:
Wait, so the `quality` when a native amount is in terms of drops. If pays is native, then you need to scale(-6) and so on.